exercises/practice/knapsack/knapsack_test.rb

```diff
@@ -0,0 +1,127 @@
+require 'minitest/autorun'
+require_relative 'knapsack'
+
+class KnapsackTest < Minitest::Test
+  Item = Data.define(:weight, :value)
+
+  def test_no_items
+    # skip
+    max_weight = 100
+    items = []
+    expected = 0
+    actual = Knapsack.new(max_weight).max_value(items)
+
+    assert_equal expected, actual,
+      "When there are no items, the resulting value should be 0."
```

Comment:
It should be or it must be?  Is this a suggestion or a requirement?

Reply:
Yeah, they "shoulda" probably not done that. ;)

Reply:
Doesn't "shoulda" sound like a tool for testing warnings and deprecation, rather for things that must happen otherwise you get failures?  But naming is hard, and the name _is_ kind of cute, definitely memorable.

Reply:
True. Too bad the gem name "muster" is already taken 😄